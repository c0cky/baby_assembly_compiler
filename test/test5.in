-391
44
5921
-53914
-16
80
59
932
768
-299
-59243
12220
0
55
48
-101
492
325
586249001834
-61029588119
4935623498
-5456345
-526238853743
10009324237
36243476
55
8922529
92018678
-199423
-65589
31
870302
-99462
-6593333
-2118154
325
85996
-26
55
6
-28
0
625
-81
-80
24
100
2350
-4967
21
