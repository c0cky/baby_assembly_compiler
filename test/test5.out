-526238853743
-61029588119
-6593333
-5456345
-2118154
-199423
-99462
-65589
-59243
-53914
-4967
-391
-299
-101
-81
-80
-28
-26
-16
0
0
6
21
24
31
44
48
55
55
55
59
80
100
325
325
492
625
768
932
2350
5921
12220
85996
870302
8922529
36243476
92018678
4935623498
10009324237
586249001834
